src/commcare_cloud/fab/operations/release.py

```diff
@@ -368,6 +372,31 @@ def record_successful_deploy():
         })
 
 
+@roles(ROLES_DEPLOY)
```

Comment:
i don't think this is necessary since it's just being called as a python function.

Reply:
I was mostly blindly matching what other nearby/similar functions are doing. Does the same apply to `record_successful_deploy` and `record_successful_release` or are they somehow different?

Reply:
`record_successful_deploy` and `record_successful_release` are executed as tasks so they need the decorator to know which hosts to run on.

Reply:
:+1: I think I'm going be doing a bit of other related work for the QA team, so will fix it when I do that.